tests/MapTest.php

```diff
@@ -1141,6 +1141,36 @@ public function testHasPath()
 	}
 
 
+	public function testIfBlankElse()
+	{
+		Map::from( ['a' => 1, 'b' => 0] )->if(
+			fn(Map $map) => $map->has('c'),
```

Comment:
`fn` is unknown in PHP 7.3 and before. Can you change that to `function` instead?

Reply:
resolved in 43ce8d41b8894a4abc84e2be81bd51237264f8ef